Packages/com.unity.inputsystem/InputSystem/Controls/ButtonControl.cs

```diff
@@ -87,8 +87,60 @@ public new bool IsValueConsideredPressed(float value)
         /// </summary>
         /// <value>True if button is currently pressed.</value>
         /// <remarks>
-        /// A button is considered press if it's value is equal to or greater
+        /// A button is considered pressed if its value is equal to or greater
         /// than its button press threshold (<see cref="pressPointOrDefault"/>).
+        /// You can use this to read whether specific keys are currently pressed by using isPressed on keys, as shown in the following examples:
+        /// <example>
+        /// <code>
+        /// // Using KeyControl property directly.
+        /// Keyboard.current.spaceKey.isPressed
+        /// Keyboard.current.aKey.isPressed // etc.
+        ///
+        /// // Using Key enum.
+        /// Keyboard.current[Key.Space].isPressed
+        ///
+        /// // Using key name.
+        /// (KeyControl)Keyboard.current["space"]).isPressed
```

Comment:
There is a left bracket missing here.

Reply:
at the start there should be two. e.g.
((KeyControl)Keyboard.current["space"]).isPressed

Reply:
with small changes like this where you're sure of the fix, you can can ahead and fix it as a changeset directly in github if you like (saves you even having to check out the branch)